lib/features/manage_account/presentation/notification/bindings/notification_binding.dart

```diff
@@ -0,0 +1,71 @@
+import 'package:get/get.dart';
+import 'package:tmail_ui_user/features/base/base_bindings.dart';
+import 'package:tmail_ui_user/features/manage_account/data/datasource/notification_datasource.dart';
+import 'package:tmail_ui_user/features/manage_account/data/datasource_impl/notification_datasource_impl.dart';
+import 'package:tmail_ui_user/features/manage_account/data/repository/notification_repository_impl.dart';
+import 'package:tmail_ui_user/features/manage_account/domain/repository/notification_repository.dart';
+import 'package:tmail_ui_user/features/manage_account/domain/usecases/get_notification_setting_interactor.dart';
+import 'package:tmail_ui_user/features/manage_account/domain/usecases/toggle_notification_setting_interactor.dart';
+import 'package:tmail_ui_user/features/manage_account/presentation/notification/notification_controller.dart';
+import 'package:tmail_ui_user/main/exceptions/cache_exception_thrower.dart';
+import 'package:tmail_ui_user/main/permissions/notification_permission_service.dart';
+
+class NotificationBinding extends BaseBindings {
+  @override
+  void bindingsController() {
+    Get.put(NotificationController(
```

Comment:
why not use `lazyPut `?

Reply:
NotificationBinding is called right before NotificationView appears, NotificationController will immediately be initialized. So `put` & `lazyPut` behave the same.